src/di.php

```diff
@@ -408,7 +408,7 @@
 $di['is_admin_logged'] = function () use ($di) {
     if (!$di['auth']->isAdminLoggedIn()) {
         $api_str = '/api/';
-        $url = $di['request']->getQuery('_url', null, '');
+        $url = $_GET['_url'] ?? null;
```

Comment:
This used to default to an empty string (`''`)

Reply:
It did - not sure if it matters here - but updated.